### PR TITLE
soroban-rpc: disable persistence support

### PR DESCRIPTION
**Pull request by jacekn**
### What

Temporarily hide persistenct support in the soroban-rpc chart.

### Why

Persistence is not fully tested at this stage so we'd like it to be less visible at this stage.

### CODE REVIEW

File: charts/soroban-rpc/values.yaml [hunk 1]
```diff
@@ -78,12 +78,6 @@ sorobanRpc:
     #annotations:
     #  cert-manager.io/cluster-issuer: "default"
 
-  ## Whether to provision persistent volume. Recommended for production use cases. Usage of the PV has benefit of avoiding ephemeral storage, which can vary on different infrastructure and potentially be too low for the combined needs of the rpc and captive core processes. Choosing a PV storage class also allows leveraging any IaaS provided volumes with solid read/write performance.
```
Comment (sreuland) [inline charts/soroban-rpc/values.yaml:81]: @jacekn , I did some checking on rpc runtime disk usage after our review meeting, and found some great benchmarking that @2opremio has done:

https://github.com/stellar/soroban-tools/issues/464

tldr, on pubnet, the container runtime will get close to ~30GB of combined disk usage for rpc and core as of now and can grow, which I think will be too close for comfort on ephemeral default limits that containers usually get on cluster. 

Can we update this section instead of removal and comment on expectations on disk space are network dependent, i.e. something like this:

> Whether to provision persistent volume(PV). Recommended for production use cases and any deployment that is configured to be on the pubnet network. On pubnet network, disk requirements start at 30GB minimum, which pushes on the typical upper limit of ephemeral storage provided in cluster containers.  Choosing a PV storage class available on from your cluster also allows choosing one that can provide a known performant read/write throughput, we recommend at least 3k IOPS, 5MB/sec.

**Comment by sreuland** [comment]
@jacekn , @tsachiherman , I tested this chart on `soroban-rpc-futurenet-dev`, this had a prior deployment of rpc direct from a manifest. First I deleted the entities from that deployment, then checked out this pr locally and installed the chart onto cluster:

```
$ helm install -f rpc_values.yml --namespace soroban-rpc-futurenet-dev dev-soroban-rpc ./charts/soroban-rpc/
```

I changed `ingress.host=rpc-futurenet.kube001-dev-test.services.stellar-ops.com` and `persistence.enabled=true` in values.yml and confirmed the deployment had the values. rpc service was operational. I manually deleted the pod, and watched the restart, it worked on persistent volume which had files from prior instance. 

looks great! I think for upcoming soroban release, as part of release steps related to first time/boot strap, we just need to manually delete the prior resources from manifest deployment in `soroban-rpc-futurenet-prd`, then ensure this `0.1.0-beta.1` chart is deployed to the stellar repo, and run helm install, likely need to build and specify a new rpc image for the preview10 tag on soroban-tools also and specify that image version in values.yml as `image.sorobanRpc.tag`

**Review by sreuland — APPROVED** [review]
